src/main/java/hudson/remoting/ChannelBuilder.java

```diff
@@ -48,9 +48,16 @@
     private static final Logger LOGGER = Logger.getLogger(ChannelBuilder.class.getName());
     private static /* non-final for Groovy */ boolean CALLABLES_CAN_IGNORE_ROLECHECKER = Boolean.getBoolean(ChannelBuilder.class.getName() + ".allCallablesCanIgnoreRoleChecker");
 
+    private static final Set<String> REMOTING_CALLABLES = new HashSet<>();
     private static final Set<String> SPECIFIC_CALLABLES_CAN_IGNORE_ROLECHECKER = new HashSet<>();
 
     static {
+        REMOTING_CALLABLES.add(RemoteInvocationHandler.RPCRequest.class.getName());
```

Comment:
Maybe we can have all of these extend some internal (restricted) base class which implements the empty `checkRoles`? Would avoid the need for this enumeration.